### PR TITLE
teleprompter: 2.3.4 -> 2.4.0

### PR DESCRIPTION
**Pull request by Scriptkiddi**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sikmir** [comment]
Please, update license to `gpl3Plus` according to
https://github.com/ImaginarySense/Imaginary-Teleprompter/blob/3a11d747493837e722a9e5a6bf9dc72fe1b936e3/package.json#L6

File: pkgs/applications/misc/teleprompter/default.nix [hunk 1]
```diff
@@ -5,11 +5,11 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "teleprompter";
-  version = "2.3.4";
+  version = "2.4.0";
 
   src = fetchurl {
-    url = "mirror://sourceforge/teleprompter-imaginary-films/imaginary-${pname}-${version}-64bit.tar.gz";
-    sha256 = "084ml2l3qg46bsazaapyxdx4zavvxp0j4ycsdpdwk3f94g9xb120";
+    url = "mirror://asourceforge/teleprompter-imaginary-films/imaginary-${pname}-${version}-64bit.tar.gz";
```
Comment (ajs124) [inline pkgs/applications/misc/teleprompter/default.nix:11]: why did you change this to `asourceforge` from `sourceforge`?

**Review by mweinelt — COMMENTED** [review]
Does this also work against a more recent electron version?

```
error: Package ‘electron-4.2.12’ in /home/hexa/git/nixos/master/pkgs/development/tools/electron/generic.nix:8 is marked as insecure, refusing to evaluate.


Known issues:
 - Electron version 4.2.12 is EOL
```

**Comment by Scriptkiddi** [comment]
@mweinelt done

**Comment by mweinelt** [comment]
Result of `nixpkgs-review pr 101844` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>teleprompter</li>
  </ul>
</details>

**Review by mweinelt — APPROVED** [review]
